Packages/Models/Sources/Models/Status.swift

```diff
@@ -8,6 +8,15 @@ public struct Application: Codable, Identifiable {
   public let website: URL?
 }
 
+extension Application {
+  public init(from decoder: Decoder) throws {
+    let values = try decoder.container(keyedBy: CodingKeys.self)
+
+    name = try values.decodeIfPresent(String.self, forKey: .name) ?? ""
+    website = try? values.decodeIfPresent(URL.self, forKey: .website)
```

Comment:
If decode cannot create a valid URL, just assign it nil.

Reply:
Example response for status

![CleanShot 2023-01-06 at 12 20 18@2x](https://user-images.githubusercontent.com/169561/211074306-cdb946c7-0436-448f-b916-96bfe03910f8.png)